src/jobs/push-to-docker.yaml

```diff
@@ -0,0 +1,28 @@
+executor: architect
+parameters:
+  image:
+    description: "Name of the docker image, without tag"
+    type: "string"
+  password_var:
+    default: "ARCHITECT_DOCKER_REGISTRY_PASSWORD"
+    description: "Environment variable name holding docker registry password"
+    type: "string"
+  registry:
+    default: "quay.io"
```

Comment:
How about this registry thing? Can we take that from the image?

Reply:
So splitting the string from image, like this ?
```
$ echo -n 'quay.io/giantswarm/aws-operator'|cut -d/ -f1
quay.io
```

Why not, no strong opinion here.